CMakeLists.txt

```diff
@@ -338,6 +338,7 @@ if(ENABLE_DBE)
   set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
   set(ENABLE_STANDALONE_CALCITE ON)
   add_definitions("-DENABLE_EMBEDDED_DATABASE")
+  add_definitions("-DDBEngine_LIBNAME=\"${CMAKE_SHARED_LIBRARY_PREFIX}DBEngine${CMAKE_SHARED_LIBRARY_SUFFIX}\"")
```

Comment:
What is this for ?Do we use DBEngine_LIBNAME somewhere?

Reply:
Yes, this is used in
https://github.com/omnisci/omniscidb/pull/647/files#diff-572dc5a7d5354cdeee74fa14ede04679a0fad4824c1153947f8d0e10e34e09e1R41

Reply:
Ah, right, my bad

Reply:
I don’t think this is necessary because different OSDependent/Unix/omnisci_path.cpp files are used for different OS.